test/NUIGLView/NUIDirectRenderingGLViewSample/DirectRenderingGLViewSample.cs

```diff
@@ -0,0 +1,402 @@
+﻿using Tizen.NUI;
+using Tizen.NUI.Components;
+using Tizen.NUI.BaseComponents;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System;
+
+namespace GLApplication
+{
+    using log = Tizen.Log;    
+    class Program : NUIApplication
+    {
+        const string lib = "libNUIDirectRenderingGLView-callbacks.so";
+        string[] IMAGE_PATH = {
+            "gallery-small-1.jpg",
+            "gallery-small-1.jpg",
+            "gallery-small-1.jpg",
+            "gallery-small-1.jpg",
+            "gallery-small-1.jpg",
+            "gallery-small-1.jpg"
+            //"gallery-small-2.jpg",
```

Comment:
Remove it, please~

Reply:
Thank you! I'll fix this resources and upload at my home (Office didn't allow to push image file to github :( )

Reply:
Fix done.

Reply:
Thank you :)